var/spack/repos/builtin/packages/aocl-libmem/package.py

```diff
@@ -55,7 +54,7 @@ class AoclLibmem(CMakePackage):
         multi=False,
     )
 
-    depends_on("cmake@3.15:", type="build")
+    depends_on("cmake@3.22:", type="build")
```

Comment:
Looks like this is because CMake does handle floating point arithmetic only integer, and our testing never hit a CPU + `lspcu` combination that returned a fractional number of MB as a cache size.   A fix is in progress.

Reply:
The issue has been addressed with a cmake patch in the most recent commit.

Reply:
Confirmed fixed using a rebuild on the same odd VM (48 cores)!